src/utils/sources-tree.js

```diff
@@ -156,7 +156,10 @@ function isDirectory(url: Object) {
   const parts = url.path.split("/").filter(p => p !== "");
 
   // Assume that all urls point to files except when they end with '/'
-  return parts.length === 0 || url.path.slice(-1) === "/";
+  // Or directory node has children
+  return parts.length === 0 ||
+    url.path.slice(-1) === "/" ||
```

Comment:
Hmm, lets remove that clause and the comment.

Do you know when we removed that requirement?
@arthur801031, @Dalimil?